src/VASL/build/module/map/ASLBoardPicker.java

```diff
@@ -452,11 +452,11 @@ private ArrayList<String> parseBoardVersionFile(InputStream metadata) throws JDO
                                 }
                             }
 
-                        } catch (Exception ex) {
+                        } catch (DateTimeParseException ex) {
                             // for the benefit of the log
-                            ex.printStackTrace();
+                            logger.warn(String.valueOf(ex));
```

Comment:
If you want no message but just the stack trace, do `logger.warn("", ex);`. You don't need to convert the exception to a string.

Reply:
OK. When I did logger.warn(ex) Intellij gave me an error and recommended the string conversion. I will update.